development.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This is the image we run in our local development docker-compose cluster
+#   it is built on top of the local production.Dockerfile
+FROM identity-idp-production
+
+# Use root for more configuration
+USER root
+
+# Re-install dev dependencies
+RUN apt-get update \
```

Comment:
These get removed from the production image, but I think it's OK to leave them in the dev image for re-use.